index.js

```diff
@@ -61,11 +83,9 @@ function arg(opts, {argv, permissive = false} = {}) {
 				}
 			}
 
-			/* eslint-disable operator-linebreak */
-			const [type, isArray] = Array.isArray(handlers[argName])
-				? [handlers[argName][0], true]
-				: [handlers[argName], false];
-			/* eslint-enable operator-linebreak */
+			const [type, isArray] = Array.isArray(handlers[argName]) ?
+				[handlers[argName][0], true] :
```

Comment:
This destructuring is unnecessary. Can you first store in a variable called `isArray` and then do the ternary for `type`?

Reply:
This isn't my code (I just removed the necessary eslint-comments), but sure!